package.json

```diff
@@ -47,13 +50,13 @@
     "lru-cache": "~2.7.3",
     "morgan": "^1.9.1",
     "ms": "^2.0.0",
+    "multer": "^1.4.5-lts.1",
     "nconf": "https://github.com/jfromaniello/nconf/tarball/master",
```

Comment:
[Snyk shows a vulnerability](https://app.snyk.io/org/auth0-dir-services/project/db89091f-ee88-4ba3-9cb9-7723b9ad6fdf/pr-check/bf6ba634-eea4-44be-9e00-b3a6ece37519#issue-SNYK-JS-NCONF-2395478) in the current version of `nconf` (`0.6.7`). 

Would it be possible to upgrade to `nconf@0.11.4`?